scripts/ci-test.sh

```diff
@@ -29,3 +29,11 @@ source ./scripts/fetch_ext_bins.sh
 fetch_tools
 setup_envs
 make generate test
+# TODO(ncdc): create a separate verify prow job
+make verify
+
+# Docker provider
+cd test/infrastructure/docker
```

Comment:
Can we print out some new lines and a bold message of what we're testing before we run these commands?

Reply:
Like "hey we're testing capd now?"

Reply:
Yeah, just a log line that we can quickly search for